src/Jackett.Common/Definitions/hdbao.yml

```diff
@@ -0,0 +1,297 @@
+      # definition. Should it be cleaned up in other definitions?
+      #
+      # time type: time elapsed (default)
+      selector: td:nth-child(4) > span[title]
+      attribute: title
+      optional: true
+      filters:
+        - name: append
+          # TODO: This is another demonstration of why a more verbose comment is
+          # valuable here. The `CST` abbreviation is used for both `-04:00` through
+          # `-06:00` in the Americas *and* `+08:00` in Asia. The more verbose comment
+          # removes ambiguity and saves contributor time.
+          args: " +08:00" # Asia/Taipei, CST
+        - name: dateparse
+          args: "yyyy-MM-dd HH:mm:ss zzz"
+    # TODO: I couldn't find an example of this when I sorted the full
+    # listing by date in both directions in the web UI of the exampale
+    # I copied. Where can I see the need for this in the example I
+    # copied so I can identify when it's needed for future
+    # definitions? Is it necessary for this definition?
+    date_added:
+      # time added
+      selector: td:nth-child(4):not(:has(span))
+      optional: true
+      filters:
+        - name: append
+          args: " +08:00" # CST
+        - name: dateparse
+          args: "yyyy-MM-ddHH:mm:ss zzz"
+    date:
+      text: "{{ if or .Result.date_elapsed .Result.date_added }}{{ or .Result.date_elapsed .Result.date_added }}{{ else }}now{{ end }}"
+    size:
+      selector: td:nth-child(5)
+    seeders:
+      selector: td:nth-child(6)
+    leechers:
+      selector: td:nth-child(7)
+    grabs:
+      selector: td:nth-child(8)
+    # TODO: There are no results for "2X", "2X 50%", and "30%" so I
+    # can't confirm their selectors but there are other examples in
+    # the example I copied of adding presumprive support for `imdbid`
+    # and `doubanid`, so I included them here.
+    #
+    # TODO: While testing, I could only find instances of "2X Free"
+    # and "ordinary" when I clicked through each of the top-level
+    # categories above, so those two are the only I could confirm by
+    # testing in the Prowlarr search UI. Is there a way I could have
+    # confirmed the rest?
+    downloadvolumefactor:
+      case:
+        img.pro_free: 0
+        img.pro_free2up: 0
+        img.pro_50pctdown: 0.5
+        img.pro_50pctdown2up: 0.5
+        img.pro_30pctdown: 0.3
+        "*": 1
+    uploadvolumefactor:
+      case:
+        img.pro_50pctdown2up: 2
+        img.pro_free2up: 2
+        img.pro_2up: 2
+        "*": 1
+    minimumseedtime:
+      # special 7 days (as seconds = 7 x 24 x 60 x 60)
+      # normal 3 days (as seconds = 3 x 24 x 60 x 60)
+      case:
+        # TODO: I found an instance of this in the example I copied by
+        # sorting by reverse date but that didn't yield an example for
+        # this indexer. See above regarding presumptive support. Am I
+        # correct to leave this in?
+        img[title="H&R"]: 604800
+        "*": 259200
```

Comment:
```suggestion
```

Nothing in the rules about H&R, and no torrents marked as it.

Reply:
Sorry I missed this. I remember reviewing the rules and forums looking for anything about H&R, I'm not sure what I did with those findings if I left this in. :-/